llvm/utils/lit/tests/shtest-cat.py

```diff
@@ -17,4 +17,8 @@
 # CHECK: # | [Errno 2] No such file or directory: 'temp1.txt'
 # CHECK: # error: command failed with exit status: 1
 
-# CHECK: PASS: shtest-cat :: cat.txt ({{[^)]*}})
\ No newline at end of file
+# CHECK: PASS: shtest-cat :: cat.txt ({{[^)]*}})
+
+# CHECK: Total Discovered Tests: 3
+# CHECK: Passed: 1 {{\([0-9]*\.[0-9]*%\)}}
+# CHECK: Failed: 2 {{\([0-9]*\.[0-9]*%\)}}
```

Comment:
Can a combination of `CHECK-NEXT` and/or `CHECK-EMPTY` be used in this test file? This helps with diagnosing issues, if the match fails.